src/tableDrawer.ts

```diff
@@ -392,9 +392,12 @@ function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
       doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
     }
   } else if (typeof cellStyles.lineWidth === 'object') {
-    doc.rect(cell.x, cursor.y, cell.width, cell.height, 'F')
 
     const sides = Object.keys(cellStyles.lineWidth)
+    if(sides.length === 4) {
+      const fillColor = cellStyles.fillColor === false ? null : typeof cellStyles.fillColor !== "string" ? 'F' : cellStyles.fillColor;
+      doc.rect(cell.x, cursor.y, cell.width, cell.height, fillColor);
+    }
     const lineWidth: any = cellStyles.lineWidth
 
     sides.map((side: string) => {
```

Comment:
is there a reason why `Array.map` is used instead `Array.forEach` ? 
Some backward compatibility?